src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -20,6 +20,7 @@
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Set;
+import javax.management.MBeanInfo;
 import javax.management.MBeanAttributeInfo;
```

Comment:
```suggestion
import javax.management.MBeanAttributeInfo;
import javax.management.MBeanInfo;
```
Again, just to fix CI.